deploy/sample/fio.yaml

```diff
@@ -12,8 +12,14 @@ parameters:
   #keyformat: "raw"
   #keylocation: "file:///home/pawan/key"
   poolname: "zfspv-pool"
-provisioner: openebs.io/zfs
+provisioner: zfs-localpv
```

Comment:
Was there a change in the provisioner name recently? or was this never working?

Reply:
in latest provisioner --provisioner flag is depricated. We have to use the driver name instead of that. So it was working before, but after changeing the image, it stopped working.